kornia/feature/laf.py

```diff
@@ -10,6 +10,9 @@
 from kornia.geometry.transform import pyrdown
 from kornia.testing import KORNIA_CHECK_LAF, KORNIA_CHECK_SHAPE
 
+if TYPE_CHECKING:
+    import numpy.typing as npt
```

Comment:
This requires numpy install, which should be optional ?

Reply:
Exactly, but since it is only type checking, which should happen in dev mode, it is not to have a problem -- same thing used in https://github.com/kornia/kornia/blob/master/kornia/utils/image.py